src/main/app.js

```diff
@@ -8,34 +8,43 @@
  * @class module:brew/app~Koa
  * @alias external:Koa
  */
+const path = require("path");
 const Koa = require("koa");
+const { createReadStream } = require('fs');
+const mount = require("koa-mount");
+const serve = require("koa-static");
 const api = require("./api");
 const logger = require("koa-logger");
-const { open } = require("./db/util");
+const {open} = require("./db/util");
 
 /**
  * @function startApp
  * @param {Number} port - The port on which the app will run
  * @param {dbUrl} dbUrl - The url at which the database resides
  * @returns {undefined}
  */
-const startApp = async ({ appPort, db: { host, port, cellar } }) => {
+const startApp = async ({appPort, db: {host, port, cellar}}) => {
     /**
      * @constant {Koa} module:brew/app~app
      */
     const app = new Koa();
 
     app.use(logger());
 
-    const db = await open({ host, port, cellar });
+    // const db = await open({ host, port, cellar });
 
-    const router = api({ db });
+    // const router = api({ db });
 
     // Hook up the api to the application
-    app.use(router.routes());
+    // app.use(router.routes());
 
     // Tell the app which methods are allowed
-    app.use(router.allowedMethods());
+    // app.use(router.allowedMethods());
+
+    app.use(async (ctx, next) => {
```

Comment:
All routes are currently forwarded to `index.html`. There's a lot of reloading going on because of this. Need to figure out a better strategy for not reloading the entire page at every route.